### DISCUSSION TITLE
Categorizations of Analyses and Outputs

### DISCUSSION THREAD

**Discussion by ASL-rmarshall** [cdisc-org/analysis-results-standard#49] [General]
As described in PBI #30 , there is a need for categorization of analyses and outputs.  Please use this discussion for questions and posting use cases.

**Comment by bhavinbusa** [comment]
From Dante: Another use case is to support internal review and validation of output and docs - Based on the example category of Top-line/result (or not), to ensure certain validation artifacts exist and are not "stale", based on prior data snapshots, etc.

**Comment by dditomma** [comment]
ICH E3 CSR structure: 
https://database.ich.org/sites/default/files/E3_Guideline.pdf
Ability to categorize and organize outputs into documents with specific structures, like ICH E3 CSR structure.
Other defined structures exist, as others noted: ISS, ISE, and [BIMO categories](https://www.fda.gov/regulatory-information/search-fda-guidance-documents/bioresearch-monitoring-technical-conformance-guide)

So reports could fit into different document structures, according to 
- category, and
- (possibly) order within category
with some categories and groups of categories being pre-defined by industry standard/convention.

**Comment by dditomma** [comment]
Select outputs assembled into doc structures like noted above may be used directly within report body, sometimes called "in text" tables, which could be another category / use case
